nixos/modules/services/torrent/transmission.nix

```diff
@@ -253,9 +253,9 @@ in
       };
 
       webHome = mkOption {
-        type = types.nullOr types.path;
-        default = null;
-        example = "pkgs.flood-for-transmission";
+        type = types.path;
+        default = "${cfg.package}/share/transmission/web";
```

Comment:
Related: since you reference the `config` attribute, you would need to define `defaultText` as well.

Reply:
Rollbacked this part